README.md

```diff
@@ -2,33 +2,46 @@
 [![npm version](https://img.shields.io/npm/v/single-spa.svg?style=flat-square)](https://www.npmjs.org/package/single-spa)
 [![Build Status](https://img.shields.io/travis/CanopyTax/single-spa.svg?style=flat-square)](https://travis-ci.org/CanopyTax/single-spa)
 
-Combine multiple SPAs into one SPA by implementing lifecycle functions. Allows you to:
-- [Trade off between frameworks](/docs/single-spa-ecosystem.md#help-for-frameworks) [without refreshing the page](/docs/child-applications.md)
+Combine multiple SPAs into one SPA by implementing lifecycle functions. This allows you to:
+- [Use multiple frameworks on the same page](/docs/single-spa-ecosystem.md#help-for-frameworks) [without refreshing the page](/docs/child-applications.md)
   (React, Angular 1, Angular 2, Ember, or whatever you're using)
-- [Lazy load code for improved initial load time](/docs/child-applications.md#load). This is baked in and requires no configuration.
-- [Ensure that no single part of the app can break everything](/docs/application-blast-radius.md).
-
-single-spa works in Chrome, Firefox, Safari, IE11, and Edge.
+- Write code using a new framework, without rewriting your existing app
+- Lazy load code for improved initial load time.
 
 ## Architectural Overview
+single-spa takes inspiration from React component lifecycles by applying lifecycles to entire applications.
```

Comment:
I don't know. I think the `-` makes capitalizing single-spa sort of hard to decide on. I'll change it to capitalize